health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/DrawerFieldComposer.js

```
@@ -0,0 +1,302 @@
+                style={{
+                  cursor: "pointer",
+                  fontWeight: "600",
+                  marginLeft: "1rem",
+                  fontSize: "1rem",
+                  color: PRIMARY_COLOR,
+                  display: "flex",
+                  gap: "0.5rem",
+                  alignItems: "center",
+                  marginTop: "1rem",
+                }}
+              >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)